tests/fixtures/model_form_exclude.py

```diff
@@ -0,0 +1,23 @@
+class User(models.ModelForm):
+    name = models.CharField(max_length=255)
+    email = models.EmailField()
+
+    class Meta:
+        model = User
+        exclude = ('name',)
+
+class User2(ModelForm):
+    name = models.CharField(max_length=255)
+    email = models.EmailField()
+
+    class Meta:
+        model = User2
+        exclude = ('name',)
+
+class User3(ModelForm):
+    name = models.CharField(max_length=255)
+    email = models.EmailField()
+
+    class Meta:
+        model = User2
+        fields == '__all__'
```

Comment:
I think that you have a extra `=` in here.

Reply:
good catch!